tests/aws/services/cloudwatch/test_cloudwatch.py

```diff
@@ -1612,6 +1610,42 @@ def test_describe_minimal_metric_alarm(self, snapshot, aws_client, cleanups):
         response = aws_client.cloudwatch.describe_alarms(AlarmNames=[alarm_name])
         snapshot.match("describe_minimal_metric_alarm", response)
 
+    @markers.aws.validated
+    def test_set_alarm_invalid_input(self, aws_client, snapshot, cleanups):
```

Comment:
Q: the test seems to fail with the old provider, do we need a `skipif` here?

Reply:
Thanks!